docs/Versions.asciidoc

```diff
@@ -42,4 +42,3 @@ ifeval::["{release-state}"!="unreleased"]
 :elasticsearch-javadoc: https://artifacts.elastic.co/javadoc/org/elasticsearch/elasticsearch/{version}
 :painless-javadoc: https://artifacts.elastic.co/javadoc/org/elasticsearch/painless/lang-painless/{version}
 endif::[]
-
```

Comment:
Not sure why this last line got deleted, don't think it matters though it may be good to undeleted it just to keep the PR strictly for those things required to bump the version.